kapitan/inputs/helm.py

```diff
@@ -272,3 +282,44 @@ def load_chart(self):
             raise HelmTemplateError(error_message)
 
         return yaml.safe_load_all(output)
+
+
+def check_data_for_b64(yml_obj):
+    """
+    check for .data in kind: Secret / ConfigMap
+    replace b64 encoded strings
+    """
+    for item in yml_obj:
+        kind = item.get("kind", None)
+        if kind and (kind == "Secret" or kind == "ConfigMap"):
+            if item.get("data", None):
+                # Option 1:
+                # item["data"] = replace_b64_refs(item["data"])
+
+                # Option 2:
+                item["stringData"] = replace_b64_refs(item["data"])
```

Comment:
Do we need to recurse through .data? Since we're only support Secrets and ConfigMaps and data is a dictionary, wouldn't it be enough to iterate through its items and replace the values? I might be missing why recursion is needed

Reply:
I am not sure. I thought that any yaml object could be very deeply nested, e.g. a dictionary with lists in it....
I talked with Danny about this, and we both agreed that recursion is needed here. But we can discuss this later in the workgroup meeting.